cli/commands/plan.go

```diff
@@ -91,6 +93,8 @@ func checkPlansResponse(response *http.Response, body []byte) error {
 		}
 	case response.StatusCode == http.StatusAlreadyReported:
 		return errors.New("Cannot execute command. Command has already been issued or the plan has completed.")
+	case response.StatusCode == http.StatusExpectationFailed:
+		return errPlanStatus417
```

Comment:
Would it make sense to just return nil in this case?

Reply:
We need to return an error to not fall through to the default HTTP error handler.

Reply:
Ah got it, ok